rust/codeql-extractor.yml

```diff
@@ -13,4 +13,24 @@ file_types:
   - name: rust
     display_name: Rust files
     extensions:
-       - .rs
+      - .rs
+options:
+  trap:
+    title: Options pertaining to TRAP.
+    type: object
+    properties:
+      compression:
+        title: Controls compression for the TRAP files written by the extractor.
+        description: >
+          This option is only intended for use in debugging the extractor. Accepted
+          values are 'brotli' (the default, to write brotli-compressed TRAP), 'gzip', and 'none'
```

Comment:
uh, do we really support brotli?

Reply:
Probably not, I guess I should have copied that snippet from ruby instead of c# ;-)

Reply:
food for thought: we could add `zstd` support, that can speed up the trap import phase (zstd is really fast at decompression).